### PR TITLE
allow to share variables between roles using a wazo_vars role

### PR DESCRIPTION
**Pull request by fredericlepied**

### CODE REVIEW

File: roles/wazo_vars/defaults/main.yml [hunk 1]
```diff
@@ -22,3 +35,5 @@ tenant_name: my-company
 api_client_name: api-client
 api_client_password: api-password
 
+wazo_debian_repo_upgrade: main
+wazo_distribution_upgrade: wazo-dev-stretch
```
Comment (sduthil) [inline roles/wazo_vars/defaults/main.yml]: The first block of variables is a bit heteroclit. Could you split the first block?

**Review by sduthil — APPROVED** [review]

File: roles/wazo_vars/defaults/main.yml (hunk 1 above)
Comment (fredericlepied) [reply roles/wazo_vars/defaults/main.yml]: done

**Comment by fredericlepied** [comment]
recheck

**Comment by fredericlepied** [comment]
recheck

**Pull request merged**